src/util/block-site.js

```diff
@@ -44,19 +44,20 @@ export const isCurrentWebsiteBlocked = () => {
     });
 };
 
-export const isCurrentTab = () => {
-    return new Promise((resolve) => {
-        if (!(window.chrome && chrome.tabs)) return resolve(true); // no chrome env.
-
-        chrome.tabs.query({ active: true, currentWindow: true }, function (tabs) {
-            let tab = tabs[0];
-
-            chrome.tabs.getCurrent(current => {
-                resolve(tab.id === current.id);
-            });
-        });
-    });
-}
+// // Currently not used..
+// export const isCurrentTab = () => {
```

Comment:
can we delete it if it's not used?

Reply:
Yes sure! Removed in b9842880283989c72fcfb38e75295b3237ee482e